src/applications/personalization/profile360/reducers/index.js

```diff
@@ -18,21 +18,21 @@ import {
 
 const editModalFormsInitialState = {
   financialInstitutionRoutingNumber: {
-    errorMessage: '',
+    errorMessage: undefined,
     field: {
       value: '',
       dirty: false,
     },
   },
   accountNumber: {
-    errorMessage: '',
+    errorMessage: undefined,
     field: {
       value: '',
       dirty: false,
     },
   },
   accountType: {
-    errorMessage: '',
+    errorMessage: undefined,
```

Comment:
`ErrorableSelect`s require a value of `undefined` for its `errorMessage` field (as opposed to any falsy value), or else the label will appear [bolded](https://github.com/department-of-veterans-affairs/veteran-facing-services-tools/blob/4ec6ee9310c2104d988a40a5d95470c7a14b2a17/packages/formation-react/src/components/ErrorableSelect/ErrorableSelect.jsx#L74). This might be worth patching in order to be consistent with the [ErrorableTextInput](https://github.com/department-of-veterans-affairs/veteran-facing-services-tools/blob/4ec6ee9310c2104d988a40a5d95470c7a14b2a17/packages/formation-react/src/components/ErrorableTextInput/ErrorableTextInput.jsx#L59).